cli/src/commands/content-directory/updateEntityPropertyValues.ts

```diff
@@ -0,0 +1,174 @@
+import ContentDirectoryCommandBase from '../../base/ContentDirectoryCommandBase'
+import { flags } from '@oclif/command'
+import { Class as ContentDirectoryClass } from '@joystream/types/content-directory'
+import inquirer from 'inquirer'
+import { CLIError } from '@oclif/errors'
+import { InputParser } from '@joystream/cd-schemas'
+
+const CONTEXTS = ['Member', 'Curator', 'Lead'] as const
+
+type CreateEntityArgs = {
+  id: string
+}
+
+type CreateEntityFlags = {
+  context: typeof CONTEXTS[number]
+}
+
+type Question = {
+  name: string
+  message: string
+  type: 'list' | 'input'
+  choices?: string[]
+  filter?: (answer: string) => string | null | number | boolean
+}
+
+export default class UpdateEntityPropertyValues extends ContentDirectoryCommandBase {
+  static description =
+    'Updates the property values of the specified entity (can be executed in Member, Curator or Lead context)'
+
+  static args = [
+    {
+      name: 'id',
+      required: true,
+      description: 'ID of the Entity',
+    },
+  ]
+
+  static flags = {
+    context: flags.string({
+      char: 'c',
+      description: 'Actor context to execute the command in (Member/Curator/Lead)',
+      required: true,
+      options: [...CONTEXTS],
+    }),
+  }
+
+  getQuestionsFromClass = (
```

Comment:
As I mentioned in the first comment - there seems to be a lot of functionality copy-pasted from `createEntity.ts` here, which is always best to avoid. Can this be moved to `ContentDirectoryCommandBase` for example?